### PR TITLE
Migration 1.6

### PR DESCRIPTION
**Pull request by fitn**
Unit tests are passing. Travis does not work due to Akeneo PIM EE

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ A bundle that extend the Akeneo PIM Enterprise CatalogRuleBundle, adding the pos
 
 | ClassificationRuleBundle | Akeneo PIM Enterprise Edition |
 |:------------------------:|:-----------------------------:|
+| v1.2.*                   | > v1.6                        |
```
Comment (jmleroux) [inline README.md]: >=

File: README.md [hunk 2]
```diff
@@ -17,7 +18,7 @@ A bundle that extend the Akeneo PIM Enterprise CatalogRuleBundle, adding the pos
 You can install the bundle with composer:
 
 ```bash
-    php composer.phar require akeneo-labs/classification-rule-bundle:1.1.*
+    php composer.phar require akeneo-labs/classification-rule-bundle:1.2.*
```
Comment (jmleroux) [inline README.md]: `~1.2`

**Comment by jmleroux** [comment]
GTM

File: README.md (hunk 2 above)
Comment (fitn) [inline README.md]: As discussed together, I'd prefer to put 1.2.* upgrades with unwanted BC ;)

**Pull request merged**